app/lib/efile/dependent_eligibility/eip_one.rb

```diff
@@ -27,6 +27,8 @@ def tax_year_2020?
       end
 
       def is_qualifying_child?
+        return dependent.qualifying_child if dependent.is_a? EfileSubmissionDependent
```

Comment:
because if it is an efile submission dependent, the qualifying_child attribute will have been set to true or false based on a previous run of the qualifying logic -- we can look to that instead of running the calculation again